services/document_conversion/v1.js

```diff
@@ -0,0 +1,120 @@
+module.exports = function(RED) {
+
+	var http = require('http');
+	var https = require('https');
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ['https' is defined but never used.](https://www.codacy.com/app/rezgui-y/node-red-node-watson/file/2392370589/issues/source?bid=3225115&fileBranchId=3225117#l4)